source/gameengine/Ketsji/KX_GameObject.cpp

```diff
@@ -236,7 +240,10 @@ void KX_GameObject::SetName(const char *name)
 
 PHY_IPhysicsController* KX_GameObject::GetPhysicsController()
 {
-	return m_pPhysicsController;
+	if (m_pPhysicsController && !m_suspendedPhysics) {
```

Comment:
When we try to set an attribute if physics controller has been removed with : 
CcdPhysicsEnvironment *env = (CcdPhysicsEnvironment *)GetScene()->GetPhysicsEnvironment();
env->RemoveCcdPhysicsController((CcdPhysicsController *)GetPhysicsController());

it causes a crash.